src/_premake_init.lua

```diff
@@ -241,6 +241,24 @@
 	}
 
 	api.register {
+		name = "debugger",
+		scope = "config",
+		kind = "string",
+		allowed = {
+			"VisualStudio",
+			"GDB",
+			"LLDB",
+		}
+	}
+
+	api.register {
+		name = "debugpathmap",
+		scope = "config",
+		kind = "list:keyed:path",
+		tokens = true,
+	}
```

Comment:
This allows to register path maps, which are common when using GDB, since DWARF often embeds absolute paths relative to the filesystem a binary was built on, and a debuggee will need to remap that directory structure to their own local source tree.